### ISSUE TITLE
No matching of current module

### ISSUE THREAD

**Issue by ChrisAlthaus** [KaihuaTang/Scene-Graph-Benchmark.pytorch#100]
Hi, I'm trying to train the underline backbone with detector_pretrain_net.py. But for approximately 50% of the weights I'm getting 'NO-MATCHING of current module'.
Is this something to fix or is this normal?

```
2020-11-16 10:03:06,350 maskrcnn_benchmark.utils.checkpoint INFO: Loading checkpoint from catalog://ImageNetPretrained/FAIR/20171220/X-101-32x8d
2020-11-16 10:03:06,350 maskrcnn_benchmark.utils.checkpoint INFO: catalog://ImageNetPretrained/FAIR/20171220/X-101-32x8d points to https://dl.fbaipublicfiles.com/detectron/ImageNetPretrained/20171220/X-101-32x8d.pkl
2020-11-16 10:03:06,350 maskrcnn_benchmark.utils.checkpoint INFO: url https://dl.fbaipublicfiles.com/detectron/ImageNetPretrained/20171220/X-101-32x8d.pkl cached in /home/althausc/.torch/models/X-101-32x8d.pkl
2020-11-16 10:03:06,763 maskrcnn_benchmark.utils.c2_model_loading INFO: Remapping C2 weights
2020-11-16 10:03:06,763 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: conv1_w               mapped name: conv1.weight
2020-11-16 10:03:06,763 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: pred_b                mapped name: fc1000.bias
2020-11-16 10:03:06,763 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: pred_w                mapped name: fc1000.weight
2020-11-16 10:03:06,763 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res2_0_branch1_bn_b   mapped name: layer1.0.downsample.1.bias
2020-11-16 10:03:06,763 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res2_0_branch1_bn_s   mapped name: layer1.0.downsample.1.weight
2020-11-16 10:03:06,764 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res2_0_branch1_w      mapped name: layer1.0.downsample.0.weight
2020-11-16 10:03:06,764 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res2_0_branch2a_bn_b  mapped name: layer1.0.bn1.bias
2020-11-16 10:03:06,764 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res2_0_branch2a_bn_s  mapped name: layer1.0.bn1.weight
2020-11-16 10:03:06,764 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res2_0_branch2a_w     mapped name: layer1.0.conv1.weight
2020-11-16 10:03:06,764 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res2_0_branch2b_bn_b  mapped name: layer1.0.bn2.bias
2020-11-16 10:03:06,764 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res2_0_branch2b_bn_s  mapped name: layer1.0.bn2.weight
2020-11-16 10:03:06,764 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res2_0_branch2b_w     mapped name: layer1.0.conv2.weight
2020-11-16 10:03:06,764 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res2_0_branch2c_bn_b  mapped name: layer1.0.bn3.bias

...

2020-11-16 10:03:06,782 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res5_2_branch2c_bn_b  mapped name: layer4.2.bn3.bias
2020-11-16 10:03:06,782 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res5_2_branch2c_bn_s  mapped name: layer4.2.bn3.weight
2020-11-16 10:03:06,782 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res5_2_branch2c_w     mapped name: layer4.2.conv3.weight
2020-11-16 10:03:06,782 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res_conv1_bn_b        mapped name: bn1.bias
2020-11-16 10:03:06,782 maskrcnn_benchmark.utils.c2_model_loading INFO: C2 name: res_conv1_bn_s        mapped name: bn1.weight
2020-11-16 10:03:06,783 maskrcnn_benchmark.utils.c2_model_loading INFO: Remapping conv weights for deformable conv weights
2020-11-16 10:03:06,821 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.bn1.bias                  loaded from layer1.0.bn1.bias            of shape (256,)
2020-11-16 10:03:06,821 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.0.bn1.running_mean of shape (256,)
2020-11-16 10:03:06,821 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.0.bn1.running_var of shape (256,)
2020-11-16 10:03:06,821 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.bn1.weight                loaded from layer1.0.bn1.weight          of shape (256,)
2020-11-16 10:03:06,821 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.bn2.bias                  loaded from layer1.0.bn2.bias            of shape (256,)
2020-11-16 10:03:06,821 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.0.bn2.running_mean of shape (256,)
2020-11-16 10:03:06,821 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.0.bn2.running_var of shape (256,)
2020-11-16 10:03:06,821 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.bn2.weight                loaded from layer1.0.bn2.weight          of shape (256,)
2020-11-16 10:03:06,821 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.bn3.bias                  loaded from layer1.0.bn3.bias            of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.0.bn3.running_mean of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.0.bn3.running_var of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.bn3.weight                loaded from layer1.0.bn3.weight          of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.conv1.weight              loaded from layer1.0.conv1.weight        of shape (256, 64, 1, 1)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.conv2.weight              loaded from layer1.0.conv2.weight        of shape (256, 8, 3, 3)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.conv3.weight              loaded from layer1.0.conv3.weight        of shape (256, 256, 1, 1)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.downsample.0.weight       loaded from layer1.0.downsample.0.weight of shape (256, 64, 1, 1)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.downsample.1.bias         loaded from layer1.0.downsample.1.bias   of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.0.downsample.1.running_mean of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.0.downsample.1.running_var of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.0.downsample.1.weight       loaded from layer1.0.downsample.1.weight of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.1.bn1.bias                  loaded from layer1.1.bn1.bias            of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.1.bn1.running_mean of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.1.bn1.running_var of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.1.bn1.weight                loaded from layer1.1.bn1.weight          of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.1.bn2.bias                  loaded from layer1.1.bn2.bias            of shape (256,)
2020-11-16 10:03:06,822 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.1.bn2.running_mean of shape (256,)
2020-11-16 10:03:06,823 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.1.bn2.running_var of shape (256,)
2020-11-16 10:03:06,823 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.1.bn2.weight                loaded from layer1.1.bn2.weight          of shape (256,)
2020-11-16 10:03:06,823 maskrcnn_benchmark.utils.model_serialization INFO: REMATCHING! backbone.body.layer1.1.bn3.bias                  loaded from layer1.1.bn3.bias            of shape (256,)
2020-11-16 10:03:06,823 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.1.bn3.running_mean of shape (256,)
2020-11-16 10:03:06,823 maskrcnn_benchmark.utils.model_serialization INFO: NO-MATCHING of current module: backbone.body.layer1.

... and so on
```

Greeting,
Christian

**Comment by JianxiangFENG** [comment]
same problem here, have you solved it?

**Comment by ChrisAlthaus** [comment]
@JianxiangFENG No I did not solved it yet. But I'm reaching an evaluation of approx. ~20-25 AP50 (here mAP) on the validation set. Since it's in the range of the authors results, I think this warning is not important. 
```
"The above pretrained Faster R-CNN model achives 38.52/26.35/28.14 mAp on VG train/val/test set respectively"
```

**Comment by DavideBuffelli** [comment]
I am having the same warning. Did anyone find a solution?

**Comment by JianxiangFENG** [comment]
> I am having the same warning. Did anyone find a solution?

In the end the training worked so I didn't pay too much attention here.
